platform/security/rest/security-rest-cxfwrapper/src/test/java/org/codice/ddf/cxf/SecureCxfClientFactoryTest.java

```diff
@@ -181,6 +186,27 @@ public void validateConduit() throws SecurityServiceException {
                 .isDisableCNCheck(), is(true));
     }
 
+    @Test
+    public void testHttpsClientWithSystemProperty() {
+        System.setProperty("property", TEST_SYSTEM_PROPERTY);
```

Comment:
Also if you remove the conditional above, you could rewrite this test to assert that the property resolver gets called instead. That way you avoid any potential fragility from modifying your environment.